recipes.html

```diff
@@ -0,0 +1,49 @@
+=<!DOCTYPE html>
```

Comment:
```suggestion
<!DOCTYPE html>
```
Getting rid of the = will get rid of the yellow bar above your navigation on the homepage